make/hotspot/lib/JvmFeatures.gmk

```diff
@@ -170,12 +170,12 @@ ifeq ($(call check-jvm-feature, link-time-opt), true)
   # later on if desired
   JVM_OPTIMIZATION := HIGHEST_JVM
   ifeq ($(call isCompiler, gcc), true)
-    JVM_CFLAGS_FEATURES += -flto=auto -fuse-linker-plugin -fno-strict-aliasing \
+    JVM_CFLAGS_FEATURES += -DINCLUDE_LTO=1 -flto=auto -fuse-linker-plugin -fno-strict-aliasing \
```

Comment:
I'm not sure why this was included in this PR (and similarly below), since
there's no use of it anywhere. However, it would be useful for
conditionalizing the definition of FORBID_C_FUNCTION, as suggested in JBS. But
just adding the defines here is not sufficient. In utilities/macros.hpp, there
should be a conditional #define to the default value (0) if the macro is not
already defined, so that we can consistently use `#if INCLUDE_LTO` and
`#if !INCLUDE_LTO`.

Reply:
I'd prefer a more meaningful name.  LTO seems a little generic and opaque.  If you know then you
know, but if you don't then you are puzzled.

Reply:
I renamed the define to INCLUDE_LINK_TIME_OPTIMIZATION.